runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -716,6 +708,49 @@ public void onResponse(Call call, Response response) throws IOException
 		});
 	}
 
+	/**
+	 * Determines the world type folder, if any, to which the
+	 * screenshot should be saved.
+	 *
+	 * @return the world type folder name as a string,
+	 * or {@code null} if the world type is a normal world
+	 * (Free, Members, PVP, High-Risk PVP)
+	 */
+	private String getWorldFolderType()
```

Comment:
Consider Optional<String> getSubdirectoryName()?

Maybe change Javadoc to something like:
/** Determines the subdirectory name (if any) that the screen should be saved. This is determined by the type of the world (Deadman, Last man standing, etc) that the player is on. */

And update the caller accordingly.

Reply:
Looks like markdown ate my comment, updated.